### PR TITLE
Added start/stop animation for `Animatable` images

### PR DESCRIPTION
**Pull request by sr01**

### CODE REVIEW

File: appintro/src/main/java/com/github/appintro/model/SliderPage.kt [hunk 1]
```diff
@@ -25,6 +26,7 @@ data class SliderPage @JvmOverloads constructor(
     var title: CharSequence? = null,
     var description: CharSequence? = null,
     @DrawableRes var imageDrawable: Int = 0,
+    var startAnimation: Boolean = false,
```
Comment (cortinico) [inline appintro/src/main/java/com/github/appintro/model/SliderPage.kt:29]: I think we should not add `startAnimation` as a boolean flag to `SliderPage` (and generally be exposed).

**Review by cortinico — CHANGES_REQUESTED** [review]
Thanks for your contrib.

I'm fine with the idea that if the provided Drawable is an Animatable, then we can animate it.

However I would suggest we don't expose a new `startAnimation` flag.

File: appintro/src/main/java/com/github/appintro/model/SliderPage.kt (hunk 1 above)
Comment (sr01) [reply appintro/src/main/java/com/github/appintro/model/SliderPage.kt:29]: Agree, its not required.

**Comment by cortinico** [comment]
@sr01 What's the status of this PR?

**Review by cortinico — APPROVED** [review]

**Pull request merged**